package cleanup

hundeboll-silicom opened the pull request.

tswhison commented on opae.spec.in:
```diff
@@ -194,6 +196,8 @@ ldconfig
 %dir @CMAKE_INSTALL_PREFIX@/src/opae
 @CMAKE_INSTALL_PREFIX@/src/opae/*
 @CMAKE_INSTALL_PREFIX@/bin/opae.io
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libboard_common.a
```
We were told not to package .a files.  It violates some sort of Fedora upstreaming guideline.

hundeboll-silicom replied:
You are right:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#packaging-static-libraries

I've pushed new/updated commits where installation of the two static libraries is disabled instead.

tswhison commented on opae.spec.in:
```diff
@@ -272,6 +275,7 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/bin/afu_sim_setup
 @CMAKE_INSTALL_PREFIX@/bin/with_ase
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libase*
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libopae-c-ase*
```
I think that as of [#116](https://github.com/OPAE/opae-libs/pull/116), the documentation will have to be updated to say that -DBUILD_ASE=yes is always given when the intention is to build RPMs.

tswhison replied:
Ah yes.. I should have looked at the preceding lines.  Please note that BUILD_ASE should be deprecated in favor of OPAE_BUILD_SIM.  Thanks

r-rojo commented on opae.spec.in:
```diff
@@ -218,6 +220,7 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libboard_n3000.so*
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libboard_d5005.so*
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libboard_a10gx.so*
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libv-opae.so*
```
This library has been renamed to libopae-v.so

hundeboll-silicom replied:
Fixed

hundeboll-silicom commented:
ping

hundeboll-silicom commented:
@r-rojo @tswhison @anandaravuri please review (again)

I've rebased on current master, and added commits to use `opae_python_install()` for opae.admin and pacsign.

Thanks.

tswhison commented on opae.spec.in:
```diff
@@ -206,13 +198,15 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/bin/bist_def.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_nlb3.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_nlb0.py
+@CMAKE_INSTALL_PREFIX@/bin/coreidle
```
Please remove coreidle.  It applies only to a legacy platform that is no longer supported.

hundeboll-silicom replied:
Done.

hundeboll-silicom commented:
@michelleyho Ping

hundeboll-silicom commented:
Rebased on current master to fix conflict from https://github.com/OPAE/opae-sdk/pull/1897

hundeboll-silicom commented:
@michelleyho @r-rojo bump?
